python/test/simple.py

```diff
@@ -74,4 +77,38 @@
 assert export_key==export_key1, "export_key must equal export_key1."
 assert sk==sk1, "sk must equal sk1."
 
+def register_with_global_server_key():
+    pwdU=b"simple guessable dictionary password"
+    cfg=opaque.PkgConfig()
+    cfg.skU=opaque.NotPackaged
+    cfg.pkU=opaque.NotPackaged
+    cfg.pkS=opaque.NotPackaged
+    cfg.idU=opaque.NotPackaged
+    cfg.idS=opaque.NotPackaged
+    ids=opaque.Ids("user", "server")
+    infos=opaque.App_Infos()
+    skS=pysodium.randombytes(crypto_scalarmult_SCALARBYTES);
+    # Uncomment the following if you compiled libopaque with -DNORANDOM -DTRACE and
+    # want the same output as register_with_global_server_key in
+    # src/tests/opaque-test.c. Also see a_randombytes in src/common.c.
+    #skS=ctypes.create_string_buffer(crypto_scalarmult_SCALARBYTES)
+    #for i in range(0, 32):
+    #    ctypes.memset(ctypes.addressof(skS) + i, i, 1)
+    pkS=pysodium.crypto_scalarmult_curve25519_base(skS);
```

Comment:
I was wondering. Why didn't `pysodium` have `crypto_scalarmult_base`?

Reply:
i guess i never added it? and noone else bothered doing a PR?